README.md

```diff
@@ -86,5 +86,5 @@ coreos:
 
 Please use the [CoreOS issue tracker][bugs] to report all bugs, issues, and feature requests.
 
-[bugs]: https://github.com/coreos/bugs/issues/new?labels=component/cloud-init
+[bugs]: https://github.com/flatcar-linux/bugs/issues/new?labels=component/cloud-init
```

Comment:
We don't have `bugs` repo, as well as the component/* label. Better to change it to https://github.com/flatcar-linux/Flatcar/issues/new?labels=kind/bug